src/js/components/Button/button.stories.js

```diff
@@ -0,0 +1,35 @@
+import React, { Component } from 'react';
+import { storiesOf } from '@storybook/react';
+
+import Button from '../Button/Button';
+import Grommet from '../Grommet/Grommet';
+
+class SimpleButton extends Component {
+  render() {
+    return (
+      <Grommet>
+        <Button label='Submit' onClick={() => {}} />
+      </Grommet>
+    );
+  }
+}
+
+class FocusedButton extends Component {
+  ref = React.createRef()
+
+  componentDidMount() {
+    setTimeout(() => { this.ref.current.focus(); }, 0);
```

Comment:
why for Anchor story we did not use setTimeout and here we are using setTimeout ?

Reply:
Good catch. We don't need the timeout. I think this was a copy/paste issue.